File: workspace_presentacion/Flotix2021/Flotix2021/ModelResponse/ServerResponseMetodoPago.cs
using Flotix2021.ModelDTO;
using System.Collections.Generic;

namespace Flotix2021.ModelResponse
{
    class ServerResponseMetodoPago
    {
        private List<MetodoPagoDTO> _listaMetodoPago;

        public List<MetodoPagoDTO> listaMetodoPago
        {
            get { return _listaMetodoPago; }
            set { _listaMetodoPago = value; }
        }

        private ErrorBean _error;

        public ErrorBean error
        {
            get { return _error; }
            set { _error = value; }
        }
    }
}

File: workspace-servidor/flotix2021/src/main/java/com/flotix/response/bean/ServerResponseMetodoPago.java
package com.flotix.response.bean;

import java.util.List;

import com.flotix.dto.MetodoPagoDTO;

/**
 * Objeto de respuesta con la informacion del metodo de pago
 * 
 * @author Flor
 *
 */
public class ServerResponseMetodoPago {

	private List<MetodoPagoDTO> listaMetodoPago = null;

	private ErrorBean error = new ErrorBean();

	public ServerResponseMetodoPago() {
		super();
	}

	public List<MetodoPagoDTO> getListaMetodoPago() {
		return listaMetodoPago;
	}

	public void setListaMetodoPago(List<MetodoPagoDTO> listaMetodoPago) {
		this.listaMetodoPago = listaMetodoPago;
	}

	public ErrorBean getError() {
		return error;
	}

	public void setError(ErrorBean error) {
		this.error = error;
	}
}
